core-rust/state-manager/src/protocol/protocol_updates/protocol_update_definition.rs

```diff
@@ -114,6 +114,7 @@ impl ProtocolStateComputerConfig {
     ) -> ExecutionConfigurator {
         let mut costing_parameters = self.costing_parameters;
         if no_fees {
+            // TODO(resolve during review): replace with `SystemOverrides.disable_costing`?
```

Comment:
Yes, definitely :). There's actually a lot of stuff we can remove / simplify with these system overrides changes.